katalog/elasticsearch-single/rules.yml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: k8s
+    role: alert-rules
+  name: es-rules
+  namespace: logging
+spec:
+  groups:
+    - name: es.rules
+      rules:
+        - alert: ElasticClusterRed
+          expr: elasticsearch_cluster_health_status{color="red"} == 1
+          for: 30m
+          labels:
+            severity: warning
+          annotations:
+            summary: 'Elastic Cluster Red (instance {{ $labels.instance }})'
+            description: "Elastic Cluster Red status\n  VALUE = {{ $value }}\n  LABELS: {{ $labels }}"
+        - alert: ElasticClusterYellow
+          expr: elasticsearch_cluster_health_status{color="yellow"} == 1
+          for: 30m
+          labels:
+            severity: warning
+          annotations:
+            summary: 'Elastic Cluster Yellow (instance {{ $labels.instance }})'
+            description: "Elastic Cluster Yellow status\n  VALUE = {{ $value }}\n  LABELS: {{ $labels }}"
+        - alert: ElasticNumberOfElasticHealthyNodes
+          expr: elasticsearch_cluster_health_number_of_nodes < number_of_nodes
```

Comment:
Where is `number_of_nodes` defined?

Reply:
Removed alert, there was no number_of_nodes defined